core/src/main/java/io/questdb/griffin/engine/table/FilterOnSubQueryRecordCursorFactory.java

```diff
@@ -76,6 +77,18 @@ public boolean recordCursorSupportsRandomAccess() {
         return true;
     }
 
+    @Override
+    public void toPlan(PlanSink sink) {
+        sink.type("FilterOnSubQuery");
+        sink.optAttr("filter", filter);
+        sink.child(recordCursorFactory);
+        sink.child(dataFrameCursorFactory);
+        //cursorFactories is empty until getCursorInstance() call 
```

Comment:
leftover?

Reply:
yep, removed .

Reply:
a commit reference would help to speed up the process